campair/campair/View/ToolDictionaryDetailView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ToolDictionarySubView.swift
+//  campair
+//
+//  Created by 이가은 on 2022/06/12.
+//
+
+import SwiftUI
+
+struct ToolDictionaryDetailView: View {
+    @State var selectedToolNumber : Int = 0
+    var body: some View {
+        VStack(alignment : .leading){
+            SelectingToolDictionaryView(selectedToolNumber: self.$selectedToolNumber)
+            
+            ScrollView(showsIndicators: false){
+                ForEach(0..<6){ index in
+                    SelectingToolView()
+                }
+            }
+            
+        }
+        .padding(.horizontal, 25)
+        .navigationTitle("캠핑페어")
+    }
+}
+
+struct SelectingToolDictionaryView: View {
+    @Binding var selectedToolNumber : Int
+    var body: some View {
+        ZStack(alignment: .bottom){
+            ScrollView(.horizontal, showsIndicators: false){
+                HStack(alignment: .top){
+                    ForEach(0..<16){ index in
+                        Button(action: {
```

Comment:
```suggestion
                        Button {
                            self.selectedToolNumber = index
                        } label: {
```
Keep closure expressions simple :)

Reply:
thank you ~